### DISCUSSION TITLE
Tasks: use current terminal working directory

### DISCUSSION THREAD

**Discussion by JosiahParry** [zed-industries/zed#8686] [Feature Requests]
I've gone ahead and given the tasks a shot and it works pretty nicely! One thing that isn't clear is how to use the `cwd` parameter. I'd like to, ideally, be able to have my runnable use the open terminal i have or the working directory. In this case my rust source is in a subdirectory. But my terminal is open to that working directory but the task is not using that path. 

Is there a way to get it to use that path? 

<img width="672" alt="image" src="https://github.com/zed-industries/zed/assets/13534508/a2ef6150-3f4f-4a28-8900-29a1a7d8274f">

**Comment by JosiahParry** [comment]
In this case, the Cargo.toml is in `src/rust/Cargo.toml`

**Reply by SomeoneToIgnore** [reply]
No such possibility yet, but here's a tracking issue:
https://github.com/zed-industries/zed/issues/8660
and that is in the works so hopefully comes soon-ish.
